### DISCUSSION TITLE
upgrade-docusaurus-to-3-6/

### DISCUSSION THREAD

**Discussion by giscus** [zxuqian/zxuqian.cn#86] [Announcements]
# upgrade-docusaurus-to-3-6/

这几天我把 Docusaurus 升级到了 3.6，主要是因为这个版本引入了 Rspack 和 SWC 等酷炫技术，把打包速度提升了好几倍。以前打包大概要一两分钟，现在只需要五六秒就搞定了！

https://zxuqian.cn/upgrade-docusaurus-to-3-6/

**Comment by Casta-mere** [comment]
3.6 打包是真快啊，震惊

**Reply by zxuqian** [reply]
> 3.6 打包是真快啊，震惊

是呀，真的好快

**Reply by Casta-mere** [reply]
能申请个友链么大佬
Castamere
https://www.castamerego.com
